manim/mobject/coordinate_systems.py

```diff
@@ -225,6 +225,42 @@ def get_axis_labels(
         )
         return self.axis_labels
 
+    def add_coordinates(
+        self, *axes_numbers: Optional[Iterable[float]], **kwargs
+    ) -> VGroup:
+        """Adds labels to the axes.
+
+        axes_numbers
+            The the numbers to be added to the axes. Use ``None`` to represent an axis without labels.
+
+        Examples
+        --------
+
+        .. code-block:: python
+
+            ax = ThreeDAxes()
+            x_labels = range(1, 11)
+            z_labels = range(2, 12, 2)
```

Comment:
Since the default x,y,z ranges of `ThreeDAxes` are (-6, 6, 1), (-5, 5, 1) and (-4, 4, 1) respectively, I think it would be better to have these ranges within the axes's ranges. 
```suggestion
            x_labels = range(-4, 5)
            z_labels = range(-4, 4, 2)
```